src/dataService/tagDataService.ts

```diff
@@ -140,35 +132,21 @@ export class TagDataService {
       .whereIn('tag', latestTags)
       .orderBy('_updatedAt', 'desc');
 
-    return tags.map(TagObjectMapper.mapDbModelToDomainEntity);
+    return tags.map(TagModel.toGraphqlEntity);
   }
 
   public async getTagsByName(names: string[]): Promise<Tag[]> {
-    const cleanTags = names.map(cleanAndValidateTag);
     const tags = await this.getTagsByUserSubQuery().andWhere(function () {
-      this.whereIn('tag', cleanTags);
+      this.whereIn('tag', names);
     });
-    return tags.map(TagObjectMapper.mapDbModelToDomainEntity);
-  }
-
-  public async getTagByName(tagName: string): Promise<Tag> {
-    const result = await this.getTagsByUserSubQuery().where(
-      'tag',
-      cleanAndValidateTag(tagName)
-    );
-    return TagObjectMapper.mapDbModelToDomainEntity(result[0]);
+    return tags.map(TagModel.toGraphqlEntity);
   }
 
-  public async getTagById(tagId: string): Promise<Tag> {
```

Comment:
Removed because there is no such thing as a tag id in the database, so it shouldn't be in the data service